PowerShellEditorServices.build.ps1

```diff
@@ -121,7 +121,7 @@ namespace Microsoft.PowerShell.EditorServices.Hosting
 task RestorePsesModules -If (-not (Test-Path "module/PSReadLine") -or -not (Test-Path "module/PSScriptAnalyzer")) {
     Write-Build DarkMagenta "Restoring bundled modules"
     Save-PSResource -Path module -Name PSScriptAnalyzer -Version "1.24.0" -Repository $PSRepository -TrustRepository -Verbose
-    Save-PSResource -Path module -Name PSReadLine -Version "2.4.1-beta1" -Prerelease -Repository $PSRepository -TrustRepository -Verbose
+    Save-PSResource -Path module -Name PSReadLine -Version "2.4.2-beta2" -Prerelease -Repository $PSRepository -TrustRepository -Verbose
```

Comment:
Can you add a comment in this task reminding devs that module updates need to be saved to the CFS feed?